api_tests/lib_sdk/wallets/bitcoin.ts

```diff
@@ -37,6 +37,23 @@ export class BitcoinWallet implements Wallet {
         private readonly bitcoinRpcClient: BitcoinRpcClient
     ) {}
 
+    public async mintToAddress(
+        minimumExpectedBalance: BigNumber,
+        toAddress: string
+    ): Promise<void> {
+        const blockHeight = await this.bitcoinRpcClient.getBlockCount();
+        if (blockHeight < 101) {
+            throw new Error(
+                "unable to mint bitcoin, coinbase transactions are not yet spendable"
+            );
+        }
+
```

Comment:
I am not quite sure, this is not new code :)

Reply:
Not really. We are mining 101 blocks in one go outside in the harness.
If the blockheight is not 101, there is a bug.

Waiting would only cause the test suite to hang for a very long time because we only mine 1 block per second.